tests/Unit/Overload/amp-lambda_or_pfe_in_main.cpp

```diff
@@ -5,13 +5,13 @@ using namespace concurrency;
 
 int main()
 {
-    auto a_lambda_func = []() restrict(amp) {        
-    };
-    
     parallel_for_each(extent<1>(1), [](index<1>) restrict(amp)
     {
        // OK. Since parallel_for_each is implemented as restrict(cpu,amp) inside
     });
-   
+
+    auto a_lambda_func = []() restrict(amp) {
```

Comment:
@AlexVlx is this a required change, or just re-formatting the codes?

Reply:
@AlexVlx  It seems this change is indeed needed. On the host side it's asking for kernel with name `_ZZ4mainEN3$_019__cxxamp_trampolineEv` but in the IR/ISA generated on the kernel side the mangled name is `_ZZ4mainEN3$_119__cxxamp_trampolineEv`.

Instead of fixing the test case, I think it's a regression. Please help look into this, and amend the comment inside the test as necessary.

To make the PR merge-able, I think it's okay to mark the test as XFAIL for now but we need to resolve this.